### ISSUE TITLE
markdown definitions list have extra whitespace with remark

### ISSUE THREAD

**Issue by gkatsev** [videojs/video.js#3882]
## Description
Currently, there doesn't seem to be a way to remove the whitespace in definitions lists in markdown like the `tight` option in remark-toc.

We should look into whether remark has an option for this otherwise write  remark plugin we can use instead.

**Comment by BrandonOCasey** [comment]
I looked into doing this with something existing (as it bothered me too), but I couldn't find anything. I think we will need a plugin for this.

**Comment by gkatsev** [comment]
Yeah, we need a plugin. I asked in their gitter and I have a few pointers of how to get started for doing this.
